Documentation/ProjectFile/process/convergence_criterion/DeltaX/c_DeltaX.md

```diff
@@ -1 +1,9 @@
-\ogs_missing_documentation
+Sets up a convergence criterion that checks only the norm of the solution update
+from one iteration to another.
+
+A relative and/or absolute tolerance value can be specified, which is applied to
+the norm of the whole solution vector, i.e., the entirety of all process
+variables.
+
+At least one of the relative or absolute tolerance has to be met in order to
+satisfy this convergence criterion.
```

Comment:
The "and/or" and "At least one" can be misunderstood. Better to underlie the sentences with equations.
At least I didn't get what happens if the abs and rel values are given but only one of them is met...